dockerfiles/Dockerfile

```diff
@@ -75,6 +75,7 @@ RUN cmake -DFLB_RELEASE=On \
     -DFLB_IN_SYSTEMD=On \
     -DFLB_OUT_KAFKA=On \
     -DFLB_OUT_PGSQL=On \
+    -DFLB_OUT_PARSEABLE=On \
```

Comment:
Is this what should be used in the cmake config? It's already defaulting to on

Reply:
feel free to suggest the best practice here. I tried to build without it but was not able to run the plugin.